addons/board/static/tests/dashboard_tests.js

```diff
@@ -746,7 +746,6 @@ QUnit.test('save two searches to dashboard', function (assert) {
 
     // Add the second filter
     $('span.fa-filter').click();
-    $('span.fa-filter').click();
```

Comment:
@len-odoo 
Just making sure.... was that double click intentional ?

Reply:
I've checked and without your commit the test does break without that line. 
So something must be changed by it (maybe in a positive way?).